upt_macports/upt_macports.py

```diff
@@ -28,6 +29,408 @@ def _render_makefile_template(self):
         template = env.get_template(self.template)
         return template.render(pkg=self)
 
+    def spdx_to_macports(self,spdx_license):
```

Comment:
I have to admit that I've never really looked into this decorators or different method types, so I'll follow your lead here (it's a good opportunity for me to learn along the way as well). 

Please correct me if I am wrong, but if this is a ```staticmethod``` doesn't that imply that it will not use ```self``` and therefore that shouldn't be an argument anymore?

Reply:
No you're right :) The whole point was to remove "self", and I failed :) This should be:

> @staticmethod
> def _spdx_to_macports(spdx_license):
> ...